src/core/reducers/addons.js

```diff
@@ -1,8 +1,40 @@
+/* @flow */
 import { ADDON_TYPE_THEME } from 'core/constants';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { AddonType } from 'core/types/addons';
+
+
+export const FETCH_ADDON = 'FETCH_ADDON';
```

Comment:
Do we have a naming pattern for action types btw? poke @tofumatt

Reply:
We have been inconsistent but we want `VERB_NOUN` so `FETCH_ADDON`, `LOAD_ADDON`, etc. makes sense.

Reply:
`FETCH_ADDON` to trigger the API call, and `LOAD_ADDON` to update the state with the API results, is that correct?

Reply:
Yup, that's what we want. And when starting things like autocomplete or a search it would probably be `START_SEARCH`, `START_AUTOCOMPLETE`

Reply:
wonderful!